### PR TITLE
Fix compilation

### PR DESCRIPTION
**Pull request by tomaka**

### CODE REVIEW

**Comment by tomaka** [comment]
Waiting for https://github.com/rust-lang/libc/pull/1

**Comment by simnalamburt** [comment]
New nightly is out, this PR is ready to be merged now.

File: macros/src/vertex.rs [hunk 1]
```diff
@@ -59,7 +59,7 @@ fn body(ecx: &mut base::ExtCtxt, span: codemap::Span,
                 .map(|(def, &(ident, _))| {
                     let ref elem_type = def.node.ty;
                     let ident_str = token::get_ident(ident);
-                    let ident_str = ident_str.get();
+                    let ident_str = ::std::ops::Deref::deref(&ident_str);
```
Comment (simnalamburt) [inline macros/src/vertex.rs]: What about `let ident_str = &*token::get_ident(ident);`?
[Reference](https://github.com/simnalamburt/glium/blob/418451eba297c3aea8d461836b2da89d4f96241c/macros/src/vertex.rs#L61)

Comment (tomaka) [inline macros/src/vertex.rs]: Fixed :)
Since I'm fixing this blindly, I went for the syntax I know will work.

**Comment by tomaka** [comment]
> The command "cargo test -j 1 --no-default-features --features "headless" --verbose" exited with 0.

Merging.

**Pull request merged**